snesapu.dll/DSP.h

```diff
@@ -66,6 +67,7 @@
 #define DSP_NOSURND 0x1000                      //Disable surround sound
 #define DSP_ENVSPD  0x2000                      //Synchronize envelope updates with speed
 #define DSP_NOPLMT  0x4000                      //Disable the maximum pitch limit
+#define DSP_NOMAIN  0x20000000                  //Disable main output, leaving only echo

```

Comment:
`0x8000` is better.
Sorry for not making it clear, low values ​​are public options and high values ​​are internal (aka hidden) options.